mantidproject/mantid
# Indirect - Corrections - Ensure Plot Current Preview plots transformed container in Container Subtraction and Apply Paalman Pings

In ```Interfaces > Indirect > Corrections```, the ```Container Subtraction``` and ```Apply Paalman Pings``` interfaces do not plot the transformed container or the result when clicking ```Plot Current Preview```.

This PR ensures the transformed container and the result are plot, when appropriate, by the ```Plot Current Preview``` feature.

**To test:**
Container Subtraction
1. Navigate to the ```Interfaces > Indirect > Corrections > Container Subtraction``` interface.
2. In the ```Sample``` field, provide the supplied irs26176* file.
3. In the ```Container``` field, provide the supplied irs26173* file.
4. Check the check-box labelled ```Scale Container by factor``` and enter 0.9 into the associated spinner.
5. Check the check-box labelled ```Shift x-values of container by adding``` and enter 0.1 into the associated spinner.
6. Click 'Run'.
7. Once the container subtraction is complete, click ```Plot Current Preview```.
8. Ensure the sample, container and result of the subtraction are plot in both the preview within the interface and the external plot.

Apply Absorption Corrections
1. Navigate to the ```Interfaces > Indirect > Corrections > Apply Absorption Corrections``` interface.
2. In the ```Sample``` field, provide the supplied irs26176* file.
3. In the ```Corrections``` field, provide the supplied *_Corrections file.
4. Check the check-box labelled ```Use Container``` and provide the supplied irs26173* file.
5. Check the check-box labelled ```Scale Container by factor``` and enter 0.9 into the associated spinner.
6. Check the check-box labelled ```Shift x-values of container by adding``` and enter 0.1 into the associated spinner.
7. Click 'Run'.
8. Once the algorithm is complete, click ```Plot Current Preview```.
9. Ensure the sample, container and result of applying the corrections are plot in both the preview within the interface and the external plot.

**Test Files:**
[corrections-test-files.zip](https://github.com/mantidproject/mantid/files/1796473/corrections-test-files.zip)



Fixes #21982 . 

**Release Notes** 
Bug fix has been added to the release notes.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
